digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/process/RoadNetworkChecker.scala

```diff
@@ -31,19 +31,24 @@ class RoadNetworkChecker(roadLinkService: RoadLinkService) {
       } else {
         val roadNumbers = roadwayDAO.getValidRoadNumbers
         val chunks = generateChunks(roadNumbers, 500)
+        roadNetworkDAO.getLatestRoadNetworkVersionId.getOrElse(
+          roadNetworkDAO.createPublishedRoadNetwork(Sequences.nextPublishedRoadNetworkId)
```

Comment:
Why do we create published road network two times? Here and at the row 47?

Reply:
If we have the initial situation when we don't have yet any published road networks, do we need to create before checking one published road network first?